Aufgabe_2/css/main.css

```diff
@@ -0,0 +1,120 @@
+body {
+    padding: 0;
+    margin: auto;
+    font-family: monospace;
+}
+
+header{
+    background-color: #589CA6;
+    padding: 20px 0 0 0;
+    position: sticky;
```

Comment:
Hier brauchst du noch die webkit Anweisung und eine top Anweisung damit das mit dem Sticky funktioniert. Wird aber nicht von allen Browsern unterstützt...
sticky ist prinzipiell eine super idee aber steckt leider noch in der Umsetzung bei den Browserentwicklern.